assets/src/domain/eventEditor/ui/datetimes/datesList/addNew/Modal.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { Modal as ModalAdapter } from '@infraUI/layout/modal';
+import { useEvent } from '@edtrServices/apollo/queries/events';
+import NewEntityModal from '@sharedUI/NewEntityModal';
```

Comment:
help me understand the purpose of `NewEntityModal`

Reply:
It's to abstract little bit of common logic between New Date and New Ticket modal, like classnames etc.